languages/zh.json

```diff
@@ -14,31 +14,31 @@
 	"How to make Tor connection work?": "如何让 Tor 连接工作？",
 	"How to use ZeroNet in Tor Browser?": "如何在 Tor Browser 中使用 ZeroNet ？",
 	"Disable always Tor mode": "禁用 Tor Always 模式",
-	"Enable Tor for every connection (slower)": "为每个连接启用 Tor （更慢）",
+	"Enable Tor for every connection (slower)": "为每个连接启用Tor(超级慢)",
 	"Tor always mode enabled, please restart your ZeroNet to make it work.<br>For your privacy switch to Tor browser and start a new profile by renaming the data directory.": "Tor Always 模式已启用，请重新打开你的 ZeroNet 来使它工作。<br>为了你的隐私，请切换到 Tor Browser 并通过重命名 data 目录来新建一份用户资料。",
 	"Your port \" + Page.server_info.fileserver_port + \" is closed, but your Tor gateway is running well.": "你的端口 \" + Page.server_info.fileserver_port + \" 处于关闭状态，但是你的 Tor 网关运行正常。",
 	"Your port \" + Page.server_info.fileserver_port + \" is closed. You are still fine, but for faster experience try open it.": "你的端口 \" + Page.server_info.fileserver_port + \" 处于关闭状态。你依然可以使用，但是为了更快的体验，请尝试打开它。",
 	"Show your masterseed": "显示你的种子密钥",
 	"Logout": "退出登录",
-	"Tor always mode disabled, please restart your ZeroNet.": "Tor Always 模式已禁用，请重新打开你的 ZeroNet 。",
+	"Tor always mode disabled, please restart your ZeroNet.": "Tor Always 模式已禁用，请重新打开你的ZeroNet。",
 	"Help to keep this project alive": "帮助这个项目",
 	"Updating to latest version...<br>Please restart ZeroNet manually if it does not come back in the next few minutes.": "正在更新到最新版本...<br>如果几分钟以后没有运行，请手动重启。",
 	"Internet Explorer is not fully supported browser by ZeroNet, please consider switching to Chrome or Firefox": "Internet Explorer 不被 ZeroNet 完整支持，请考虑切换到 Chrome 或 Firefox",
 	"Unsupported browser": "不支持的浏览器",
-	"New ZeroNet version: ": "新 ZeroNet 版本: ",
+	"New ZeroNet version: ": "新ZeroNet版本: ",
 	"User: ": "用户: ",
 
-	"Welcome to ZeroNet": "欢迎使用 ZeroNet",
-	"Let's build a decentralized Internet together!": "让我们共同构建一个分布式的互联网！",
-	"This site currently served by ": "这个站点目前由 ",
-	" peers, without any central server.": " 个节点驱动，没有任何中心化的服务器。",
+	"Welcome to ZeroNet": "欢迎加入 ZeroNet",
+	"Let's build a decentralized Internet together!": "让我们共创一个分布式网络吧！",
+	"This site currently served by ": "本站点目前可发现 ",
```

Comment:
"This site currently served by " should be replaced too @shortcutme (also in js)

Reply:
Yes, it should be "This site is currently served by".  @shortcutme